CHANGELOG.md

```diff
@@ -9,6 +9,15 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 [Unreleased]: https://github.com/uploadcare/uploadcare-widget/compare/v3.7.9...HEAD
 
+## [3.8.1]
+
+### Fixed
+
+* Escaped filename. See [#643][643]
```

Comment:
Escaping special symbols in filenames to avoid XSS attacks. See [#643][643].

Reply:
thanks